src/transport/PeerConnectionState.h

```diff
@@ -127,24 +114,19 @@ class PeerConnectionState
         return mReceiverSecureSession.Decrypt(input, input_length, output, header, mac);
     }
 
-private:
-    enum class MsgCounterSyncStatus
-    {
-        NotSync,
-        SyncInProcess,
-        Synced,
-    } mMsgCounterSynStatus;
+    /* GetSessionMessageCounter is only accessible by the MessageCounterManager */
```

Comment:
This comment is false, if it's trying to be a statement of fact.  Is it trying to say "should only be accessed" or something like that?

Reply:
This is a legacy comment, removed